packages/auth/src/app.ts

```diff
@@ -210,7 +210,18 @@ export class App {
           } else if (path.includes('token')) {
             route = accessTokenRoutes[tokenMethodToRoute[method]]
           } else if (path.includes('introspect')) {
-            route = accessTokenRoutes.introspect
+            this.publicRouter[method](
+              toRouterPath(path),
+              createValidatorMiddleware<ContextType<typeof route>>(openApi, {
+                path,
+                method
+              }),
+              this.config.introspectionHttpsig
+                ? signatureService.tokenHttpsigMiddleware
+                : (ctx, next) => next(),
+              accessTokenRoutes.introspect
+            )
+            continue
```

Comment:
So why did it check the signature prior to this change?

Reply:
The endpoint's `route` was previously being added to the router at the end of the `for` loop

Reply:
Of course. 👍